packages/react-scripts/config/webpack.config.js

```diff
@@ -71,6 +71,9 @@ const crossOriginLoading = bpkReactScriptsConfig.crossOriginLoading || false;
 const sriEnabled = bpkReactScriptsConfig.sriEnabled || false;
 const supressCssWarnings = bpkReactScriptsConfig.ignoreCssWarnings || false;
 
+const ssrEnabled = bpkReactScriptsConfig.ssrEnabled || false;
+const isSsrJsExisted = fs.existsSync(paths.appSsrJs);
```

Comment:
I don't think this is needed I will dig it out but the system already has build in place to detect it and work for SSR

Reply:
Have changed this part, the functionality here is to confirm it is a SSR build and it should use `build/web` as output.